form-editor-cae/src/main/java/com/tallence/formeditor/cae/elements/FormElement.java

```diff
@@ -76,9 +76,9 @@
   /**
    * Checks, if the dependent field's value matches this field's visibility config.
    * @see AdvancedSettings#getDependentElementId()
-   * @param allElements
+   * @param allElements the other fields, which might contain one dependant field
```

Comment:
```suggestion
   * @param allElements the other fields, which might contain a dependant field
```
Oder Plural, wenn das auch mehrere sein können..